lib/pinga-core/src/job/consumer.rs

```diff
@@ -0,0 +1,10 @@
+use si_id::{
```

Comment:
It looks like this is dead code because of the above file in pinga-core?

Reply:
Correct, fixed